File: CheckMyStar.Angular/src/app/www/20_Models/BackOffice/Society.model.ts
import { AddressModel } from "../Common/Address.model";

export interface SocietyModel {
    identifier: number,
    name: string,
    email: string,
    phone: string,
    logoPath?: string,
    siretCode?: string,
    vatNumber?: string,
    legalInformation?: string,
    address: AddressModel,
    isActive: boolean
}

File: CheckMyStar.Bll.Models/SocietyModel.cs
namespace CheckMyStar.Bll.Models
{
    public class SocietyModel
    {
        public int Identifier { get; set; }
        public string Name { get; set; } = string.Empty;
        public string? LogoPath { get; set; }
        public string? Email { get; set; }
        public string? Phone { get; set; }
        public string? SiretCode { get; set; }
        public string? VatNumber { get; set; }
        public string? LegalInformation { get; set; }
        public AddressModel? Address { get; set; }
        public bool IsActive { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime? UpdatedDate { get; set; }
    }
}